packages/core/data-transfer/src/strapi/__tests__/register.test.ts

```diff
@@ -20,17 +21,19 @@ jest.mock('../remote/handlers', () => ({
 }));
 
 describe('Register the Transfer route', () => {
-  test('registers the /transfer route', () => {
+  test('registers the transfer route', () => {
     const strapi = strapiMockFactory();
 
     register(strapi);
-    expect(strapi.admin.routes.push).toHaveBeenCalledWith({
-      method: 'GET',
-      path: '/transfer',
-      handler: createTransferHandler(),
-      config: {
-        auth: false,
-      },
-    });
+    expect(strapi.admin.routes.push).toHaveBeenCalledWith(
+      expect.objectContaining({
+        method: 'GET',
+        path: TRANSFER_URL,
+        handler: createTransferHandler(),
+        config: {
+          policies: expect.any(Array),
```

Comment:
we're just testing registration of the route, the actual policy checked might change but we at least know there should be a policy and not be an unrestricted route now